site/source/docs/tools_reference/emcc.rst

```diff
@@ -282,6 +282,9 @@ Options that are modified or new in *emcc* are listed below:
 ``--exclude-file <name>``
 	Files and directories to be excluded from :ref:`--embed-file <emcc-embed-file>` and :ref:`--preload-file <emcc-preload-file>`. Wildcards (*) are supported.
 
+``--fully-prepare-files``
```

Comment:
Yeah, hard to find a good name here. I used "prepare" to keep consistent with `emscripten_async_prepare()`. If we want another name, we should go and rename that as well, but I'm not sure it's worth it.

But I do agree that "fully" sounds misleading, now that you mention it. How about `--extra-prepare`?